docs/topic-guides/system-concepts/scheduling.txt

```diff
@@ -176,5 +178,10 @@ contain the following:
       pod_spec:
          metadata:
             labels:
-               pod-group.scheduling.sigs.k8s.io/name: determined
-               pod-group.scheduling.sigs.k8s.io/min-available: "2"
+               pod-group.scheduling.sigs.k8s.io/name: <unique task name>
+               pod-group.scheduling.sigs.k8s.io/min-available: <# of GPUs required>
+         spec:
+            schedulerName: coscheduler
+
+You can also use ``schedulerName: default-scheduler`` to use the default
```

Comment:
question: I think I may be misunderstanding, but this is a Kubernetes value and not a master config value right?

Reply:
yeah you got it - the `values.yaml` is from the helm chart.